plugin.xml

```diff
@@ -23,6 +24,10 @@
             <clobbers target="RemotePackage" />
         </js-module>
 
+        <js-module src="bin/www/syncStatus.js" name="syncStatus">
+            <clobbers target="SyncStatus" />
```

Comment:
Yes it's pretty common to register global types in plugins. We did the same with LocalPackage and RemotePackage types.